src/DOMUpdates.js

```diff
@@ -49,9 +42,50 @@ const domUpdates = {
         element.classList.add('hidden');
       }
     })
-  }
+  },
+
+  updateWelcomeMessage(customer = {name: '- uh oh. Looks like we had an error'}) {
+    const welcomeMessage = document.querySelector('#welcome-message');
+    welcomeMessage.innerHTML = `Welcome ${customer.name}`;
+  },
+
+  displayCustomerExpenditures(customer = {totalExpenditures: 0}) {
+    const customerExpenditure = document.querySelector('#customer-expenditure');
+    customerExpenditure.innerHTML = `Your total for all bookings is ${customer.totalExpenditures}`
+  },
 
+  populateCustomerBookings(bookedRooms = []) {
+    const customerBookings = document.querySelector('#customer-bookings');
+    bookedRooms.forEach(bookedRoom => {
+      customerBookings.innerHTML += `
+      <section role="figure">
+        <p role="heading">On ${bookedRoom.dateBooked}:</p>
+        <p>Room ${bookedRoom.number}</p>
+        <p>${bookedRoom.roomType}</p>
+      </section>
+      `
+    })
+  },
 
+  displayTodaysDate() {
+    const todaysDate = document.querySelector('#todays-date');
+    todaysDate.innerHTML = `${moment().format('MM/DD/YYYY')}`;
+  },
+
+  displayRoomsAvailableForDay(numberOfRooms = 0) {
+    const totalRoomsAvailable = document.querySelector('#total-rooms-available');
+    totalRoomsAvailable.innerHTML = `${numberOfRooms} rooms are still available for today.`;
+  },
+
+  displayTotalRevenueForDay(revenueForDay = 0) {
+    const totalRevenue = document.querySelector('#total-revenue');
+    totalRevenue.innerHTML = `Total revenue for the day so far is $${revenueForDay}.`;
+  },
+
+  displayOccupationPercentageForDay(percentage = 0) {
+    const occupationPercentage = document.querySelector('#occupation-percentage');
+    occupationPercentage.innerHTML = `${percentage}% of rooms are occupied for today.`;
+  }
```

Comment:
Trying to figure out how to best refactor these into one function - got any ideas? I used switch statements in a similar way with the last project but that ultimately felt pretty clunky.

Reply:
I've been using the value in a `querySelector` to set the conditional of an output - so: 
```
// function (value = 0, location) {
// const DOMlocation = document.querySelector(location)
// if (location == revenue) 
// DOMlocation.innerHTML = `Total Rev ${value}
// else if {_etc._}
```
might be a little clunky, but ultimately it saves some lines